src/main/scala/no/ndla/articleapi/model/domain/Article.scala

```diff
@@ -16,8 +16,8 @@ import org.json4s.FieldSerializer._
 import org.json4s.native.Serialization._
 import scalikejdbc._
 
-case class Article(id: Option[Long],
-                   revision: Option[Int],
+case class Article(_id: Option[Long],
+                   _revision: Option[Int],
```

Comment:
Jeg er også enig i at dette bare ble rot og ganske tullete. Stemmer heller for en tilbakerulling til slik det var før